guillotina/tests/test_annotation.py

```diff
@@ -0,0 +1,43 @@
+from guillotina.content import create_content_in_container
```

Comment:
I think we already have a `test_annotations.py` file -- maybe just add it here?

Or, the other bucket field tests are in `test_serialize.py`